src/js/serial_backend.js

```diff
@@ -803,7 +803,7 @@ function reinitialiseConnection(originatorTab, callback) {
             if (callback) {
                 callback();
             }
-        }, 100);
+        }, 200);
```

Comment:
What is this change for?

Reply:
Have to check again

Reply:
any out of the scope of this PR, so removed

Reply:
Maybe it had to do with the very intermittent never ending loop on the pidtuner tab.